### ISSUE TITLE
Unknown option: diamond_path Unknown option: diamond

### ISSUE THREAD

**Issue by mdayii** [Gaius-Augustus/BRAKER#307]
Hi,
I am running braker2 and I found an error in `errors/aa2nonred.stderr `directory (show below). 

```
Unknown option: diamond_path
Unknown option: diamond
```

diamond is shown as option at a script (`aa2nonred.pl `) below. But no diamond option is found at the script. A part of braker.log file that contains diamond is given below.
```
# Wed Dec  2 21:40:57 2020: Found environment variable $DIAMOND_PATH. Setting $DIAMOND_PATH to /home/user10//applications/diamond/
perl /lustre7/home/lustre4/user10/applications/augustus-3.3.2/scripts/aa2nonred.pl /lustre7/home/lustre4/user10/braker/traingenes.good.fa /lustre7/home/lustre4/user10/braker/traingenes.good.nr.fa --DIAMOND_PATH=/home/user10/applications/diamond/ --cores=1 --diamond 1> /lustre7/home/lustre4/user10/braker/aa2nonred.stdout 2>/lustre7/home/lustre4/user10/Braker/braker/errors/aa2nonred.stderr
```

**Comment by tomasbruna** [comment]
Hello,

can you check whether you are working with the latest version of the `aa2nonred.pl` script? It looks like you are using an older Augustus v3.3.2 in which the `diamond` option is not implemented.

I recommend updating both BRAKER and AUGUSTUS from their master branches on GitHub to make sure you are working with the latest version of both tools. Many errors have been recently fixed in both.

Best,
Tomas

**Comment by KatharinaHoff** [comment]
I agree with Tomas. This will likely resolve by obtaining code from github with git clone.

Btw, we are finally working towards a novel Augustus release these days...

**Comment by mdayii** [comment]
Thank you for your help. I think it is better to wait a novel Augustus release to update.

**Comment by KatharinaHoff** [comment]
Usually it is dangerous to wait for an AUGUSTUS release, because they come so rarely. But the latest one appeared today: https://github.com/Gaius-Augustus/Augustus/releases/tag/v3.4.0
If problems with Augustus scripts arise again (which is likely because we constantly develop BRAKER and Augustus), then please clone Augustus from github. The release should be fine for today, though ;-)
